src/components/PasswordInput/PasswordInput.tsx

```diff
@@ -7,9 +7,11 @@ import { invisibleIcon, visibleIcon } from "../Input/inputIcons";
 import InputConfigsProps from "src/types/InputConfigs";
 
 interface PasswordInputProps extends ConfigProps {
-  onChange: Function;
-  onBlur: Function;
-  error: any;
+  onChange?: Function;
```

Comment:
Please don't use Fuction type if it is possible

Reply:
Done.